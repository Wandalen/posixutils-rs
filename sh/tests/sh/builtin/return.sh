f () {
  echo correct
  return
  echo wrong
}

f
echo $?

g () {
  echo correct
  return 2
  echo wrong
}

g
echo $?

cat >$TEST_WRITE_DIR/return.txt <<end
echo correct
return
echo wrong
end

. $TEST_WRITE_DIR/return.txt
echo $?
rm $TEST_WRITE_DIR/return.txt