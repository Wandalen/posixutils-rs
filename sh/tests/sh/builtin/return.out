correct
0
correct
2
correct
0
